### PR TITLE
Page admin sans affichager dynamique

### PR DESCRIPTION
**Pull request by RMayeux**

### CODE REVIEW

**Review by clemcx — APPROVED** [review]
approved by the srcum master assistant

File: src/Controller/AdminController.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+class AdminController extends AbstractController
+{
+    /**
+     * @Route("/admin", name="admin")
+     */
+    public function index()
+    {
+        return $this->render('admin/index.html.twig', [
+            'controller_name' => 'AdminController',
```
Comment (benj-san) [inline src/Controller/AdminController.php]: I am pretty sure you do not need this twig variable

File: templates/admin/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,207 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello AdminController!{% endblock %}
```
Comment (benj-san) [inline templates/admin/index.html.twig]: Like wut ?

File: templates/admin/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+{% block body %}
+
+    <section id="filterContainer" class="hidden">
+    </section>
+
+    <section class="modalDescription hidden">
+        <h3>Demande n°</h3>
```
Comment (benj-san) [inline templates/admin/index.html.twig]: h1 then h2 then h3 ...

File: templates/admin/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+                                    <h4 class="card-title">Demande n°</h4>
+                                    <p> Entreprise :</p>
+                                </div>
+
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+
```
Comment (benj-san) [inline templates/admin/index.html.twig:65]: So could you tell me which bootstrap random template did you copypasted ?
I am not against bootstrap, may be usefull but please make a nice skeleton

File: templates/admin/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+        <div class="new-demande">
+            <div id="button-newdemand" href="" >Nouvelle demande</div>
+        </div>
+        <section id="demandeFormulaire" class="hidden">
+            <div id="demandeFormulaireContent">
+                <h3>Créer une demande</h3>
+                <form>
+                    <input type="text" placeholder="Nom entreprise" class="formDemande"/>
+                    <br/>
```
Comment (benj-san) [inline templates/admin/index.html.twig]: No label here for the text nor the needs(textarea)

File: templates/base.html.twig [hunk 6]
```diff
@@ -5,6 +5,7 @@
         <title>{% block title %}Welcome!{% endblock %}</title>
```
Comment (benj-san) [inline templates/base.html.twig]: title not relevant

**Review by benj-san — COMMENTED** [review]
Please fix the issues where I put the comments before I merge

**Pull request merged**